webui/react/src/components/Bar.module.scss

```diff
@@ -3,20 +3,36 @@
   display: flex;
   min-height: 2rem;
 
+  ol,
+  li {
+    margin: 0;
+    padding: 0;
+  }
+  ol {
+    display: flex;
+    flex-direction: row;
+  }
+  li {
+    display: block;
+    height: 100%;
+    margin-left: 1px;
+  }
+  li:first-child {
+    margin-left: 0;
+  }
   .bar {
     background-color: var(--theme-colors-monochrome-12);
-    border-radius: 0;
+    border-radius: 3px;
```

Comment:
**nit:** 0.3rem for consistency sake? But for modern browsers px is the preferred way to go... could do a big refactor to change everything to `px` lol

Reply:
> But for modern browsers px is the preferred way to go.

I didn't know that. sg